src/coincontrol.h

```diff
@@ -14,7 +14,7 @@
 class CCoinControl
 {
 public:
-    CTxDestination destChange;
+    CTxDestination destChange = CNoDestination();
```

Comment:
why has this been edited to CNoDestination()?

Reply:
It's just initialization; and is matching the destination default used elsewhere (including what it's set to when calling SetNull())

However from a style standpoint; it is the only one initialized in the public declaration, where everything else is initialized with SetNull().